main.cpp

```diff
@@ -32,28 +32,286 @@ Purpose:  This project continues developing Project3.
  
  */
 
+#include <iostream>
 /*
  copied UDT 1:
  */
+struct Shoe
+{
+    std::string brand { "Nike" };
+    double size { 0 };
+    std::string color { "white" };
+    std::string condition { "good" };
+    std::string securingMethod { "velcro" };
+    double percentOfSoleLeft = 100;
+    bool beingWorn = false;
+    
+    Shoe();
+    ~Shoe()
+    {
+        std::cout << "Shoe Destroyed\n";
+    }
+
+    void putOn ( std::string firstShoe = "R", bool leftFirst = true);
+    void takeOff ( std::string firstShoe = "R", bool leftFirst = true );
+    void throwAway ( bool donate = false );
+    void wearDown(int timesWorn);
+};
+
+Shoe::Shoe(){}
+
+void Shoe::putOn(std::string firstShoe, bool leftFirst) 
+{
+    if (leftFirst)
+    {
+        firstShoe = "L";
+    }
+}
+
+void Shoe::takeOff(std::string firstShoe, bool leftFirst) 
+{
+    if (leftFirst)
+    {
+        firstShoe = "L";
+    }
+}
 
+void Shoe::throwAway(bool donate) 
+{
+    if (condition == "good")
+    {
+        donate = true;
+    }
+    
+}
+
+void Shoe::wearDown(int timesWorn)
+{
+    for (int i = 0; i <= timesWorn; i += 1)
+    {
+        percentOfSoleLeft -= 0.2;
+    }
+
+    std::cout << "Sole percent left " << percentOfSoleLeft << std::endl;
+}
 /*
  copied UDT 2:
  */
+struct Wurlitzer
+{
+    int volumeLevel;
+    double vibratoAmount;
+    std::string model;
+    std::string condition;
+    int numKeys;
+    int output = 23;
+    bool on;
+    int tuning = 440;
+
+    Wurlitzer();
+    ~Wurlitzer()
+    {
+        std::cout << "Wurlitzer Destroyed\n";
+    }
```

Comment:
0) move all of your implementations of all functions to OUTSIDE of the class. 
- [ ] revise